software/contrib/traffic.md

```diff
@@ -0,0 +1,52 @@
+# Traffic
+
+This script is inspired by [Jasmine & Olive Tree's Traffic](https://jasmineandolivetrees.com/products/traffic).
+Instead of 3 trigger inputs, this version only has 2.
+
+Traffic has 3 output channels (`cv1`-`cv3`), which output a CV signal when a trigger is received on either
+of the two inputs.  The value of the output signal depends on:
+1. which input the trigger was received on and
+2. the gains for that trigger on each channel.
+
+For example, suppose the gains for channel A are set to `[0.25, 0.6]`. Whenever a trigger on `din` is received,
```

Comment:
I'd add `din` (Trigger input 1) to match the reference to Trigger input 1 in the section below.

Reply:
Done